roles/osasinfra/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_osasinfra"
+cifmw_osasinfra_artifacts_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}/tests/osasinfra"
+cifmw_osasinfra_ansible_artifacts_dir: "artifacts"
+cifmw_osasinfra_installer_artifacts_dir: "ostest"
+cifmw_osasinfra_client_pod_name: "shiftstackclient"
+cifmw_osasinfra_client_pod_namespace: "openstack"
+cifmw_osasinfra_client_pod_image: "quay.io/shiftstack-qe/shiftstack-client:latest"
+cifmw_osasinfra_shiftstack_qa_repo: "https://github.com/shiftstack/shiftstack-qa"
```

Comment:
I think it's safer to clone "https://review.gerrithub.io/shiftstack/shiftstack-qa" instead

Reply:
Done